proposal/Hydr8O/README.rst

```diff
@@ -0,0 +1,143 @@
+Basic information
+=================
+
+Project link:
+-------------
+https://github.com/Hydr8O/nsd-project
+
+Problem to solve
+================
+
+Problem Description
+-------------------
+Many companies in various industries store data about different procedures in a structured way for future use.
+
+However, there are a big chunk of unstructured data in the form of paper documents that is needed to be converted into structured format manually.
+
+The goal of this project is to automate manual data retrieval from the paper sources.
```

Comment:
Is it correct that you are trying to make a tool for OCR?

Reply:
Yes, this is correct. 
However, I'm not sure if it will be a versatile tool or a tool for some specific use case.

Should I add something to the proposal or change some parts?

Reply:
If you make it clearer that you are doing OCR, it will help others to better understand what you want to do.  But so far the proposal is clear for implementation.

The clearer the plan is the more information you may receive with discussions.  Use your own judgement.